docs/static/spec/rasa.yml

```diff
@@ -1407,19 +1451,40 @@ components:
           type: integer
           description: Time to wait between pulls from model server
 
-    JSONTrainingRequest:
+    YAMLTrainingRequest:
       type: object
       properties:
-        domain:
-          $ref: '#/components/schemas/DomainFile'
-        config:
-          $ref: '#/components/schemas/ConfigFile'
+        intents:
+          type: array
+          description: All intent names and properties
+          items:
+            $ref: '#/components/schemas/IntentDescription'
+        responses:
+          description: Bot response templates
+          type: object
+          additionalProperties:
+            $ref: '#/components/schemas/TemplateDescription'
+        session_config:
+          description: Session configuration options
+          type: object
+          properties:
+            session_expiration_time:
+              type: integer
+            carry_over_slots_to_new_session:
+              type: boolean
+        pipeline:
```

Comment:
Why the change from `#/components/schemas/ConfigFile`?

Reply:
@joejuzl Similar to domain, request body is flattened, `config` key doesn't exist anymore.

Reply:
is there no way to include the contents of `#/components/schemas/ConfigFile` at the top level, i.e. without it being under it's own key?

Reply:
I've tried using `allOf` but couldn't get it to work. To test this, I've run `make livedocs` in top level and then added:
```
    TrainingSchema:
      allOf:
        - $ref: '#/components/schemas/DomainFile'
        - $ref: '#/components/schemas/ConfigFile'
```
to the bottom of the file and referenced it in requestBody as:
```
          application/yaml:
            schema:
              - $ref: '#/components/schemas/TrainingSchema'
```
I don't see schema and only see `any` in the docs output in that case. @joejuzl @melindaloubser1 do you know a better way?

Reply:
Well that second part's schema should not be a list:
```
         application/yaml:
            schema:
              $ref: '#/components/schemas/TrainingSchema'
```
Notice the dash is missing. This is still not showing the schema, but instead of `any`, I get `string (TrainingSchema)`

Reply:
Unfortunately I haven't found another way either - Only experimented briefly but ran into the same issues you did @tayfun